submissions/TaniaBondarenko/popup2_task/style.css

```diff
@@ -0,0 +1,101 @@
+/*Common styles*/
+.navigation {
+  background-color: #fc7e08;
+  text-align: end;
+  height: 35px;
+  padding: 5px 5px 15px 5px;
+}
+
+.popup_icon_image {
+  height: 35px;
+  width: 35px;
+}
+.nav_icon {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.nav_icon_image {
+  height: 28px;
+  width: 25px;
+}
+.nav_icon:hover {
+  cursor: pointer;
+  background: rgb(175, 172, 172);
+}
+
+.popup :hover {
+  cursor: pointer;
+  background: rgb(175, 172, 172);
+}
+
+span:hover {
+  cursor: pointer;
+  background: #fc7e08;
+  border: 1px solid #fc7e08;
+}
+
+.li_icon_image {
+  width: 37px;
+  height: 35px;
+}
+
+/*Styles for pressing menu-button*/
+ul {
+  display: none;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  margin-right: 2px;
+  margin-left: auto;
+  padding: 10px 10px 10px 10px;
+}
+#checkbox,
```

Comment:
Avoid using ids for styling elements. It's better to use classes for it

Reply:
Thank you.
 And a question:
This is <input> element, which has id by default. 
Isn't it simplify code, by using this id, without creating new class?

Reply:
`input` tag doesn't have `id='checkbox'` by default. More info [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input). Maybe you mean `type='checkbox'.
So, you can use class instead of id here

Reply:
I meant that id is needed here to chain <input> with <label>. So if I have id already why need I use class